Services/UI/OJS.Services.Ui.Models/Contests/ContestServiceModel.cs

```diff
@@ -59,6 +59,10 @@ public class ContestServiceModel : IMapExplicitly
 
     public ICollection<ContestProblemServiceModel> Problems { get; set; } = null!;
 
+    public ICollection<LecturerInContestServiceModel> LecturersInContests { get; set; } = null!;
+
+    public ICollection<LecturerInContestCategoryServiceModel> LecturerInContestCategory { get; set; } = null!;
```

Comment:
Be careful not to return these into the response.

Reply:
The response model does not contain such info.